NWNXLib/Hooking/FunctionHook.cpp

```diff
@@ -21,7 +21,7 @@ FunctionHook::FunctionHook(const std::string& hookName, uintptr_t originalFuncti
     uintptr_t newFunction, const RetSizeType length)
     : m_oldFunction(), m_trampoline(), m_length(length)
 {
-    assert(m_length >= MIN_RET_SIZE && m_length < MAX_LENGTH);
+    ASSERT(m_length >= MIN_RET_SIZE && m_length < MAX_LENGTH);
```

Comment:
Preexisting, but && shouldn't be used in asserts. When it fails, since it doesn't have an explicit message, you won't know if it was too small or too big.